installer-resources/tools/uploadghasset.rb

```diff
@@ -1,32 +1,22 @@
 #!/usr/bin/env ruby
 
-if ARGV.length < 2
-  abort "\nError: The tag and file name are required. Aborting...\n\n"
+if ARGV.length < 4
+  abort "\nUsage upload_asset.rb $GH_USER $GH_PROJECT $TAG $FILE_NAME\n\n"
 end
 
 require 'octokit'
- 
-USER='getlantern'
-PROJECT='flashlight-build'
 
-TAG=ARGV[0]
-FILE_NAME=ARGV[1]
+USER=ARGV[0]
+PROJECT=ARGV[1]
+TAG=ARGV[2]
+FILE_NAME=ARGV[3]
 
 @client = Octokit::Client.new(:access_token => ENV['GH_TOKEN'])
 
-
-def release(file, tag) 
+def release(file, tag)
   releases = @client.releases "#{USER}/#{PROJECT}"
   target_release = releases.select { |r| r.tag_name == tag }[0]
 
-  assets = @client.release_assets(target_release.url)
```

Comment:
Code moved to createrelease.rb, deleting assets at this point was causing only the asset added at the end to be shown.